graphql/resolvers.go

```diff
@@ -254,6 +257,58 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, sortCatego
 	return testPointers, nil
 }
 
+func getGroupedFiles(ctx context.Context, name string, taskID string, execution int) (*GroupedFiles, error) {
+	taskFiles, err := artifact.GetAllArtifacts([]artifact.TaskIDAndExecution{{TaskID: taskID, Execution: execution}})
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, err.Error())
+	}
+	strippedFiles := artifact.StripHiddenFiles(taskFiles, gimlet.GetUser(ctx))
+	apiFileList := []*restModel.APIFile{}
+	for _, file := range strippedFiles {
+		apiFile := restModel.APIFile{}
+		err := apiFile.BuildFromService(file)
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, err.Error())
+		}
+		apiFileList = append(apiFileList, &apiFile)
+	}
+	return &GroupedFiles{Name: &name, Files: apiFileList}, nil
+}
```

Comment:
resolvers file is already growing quite big. functions that are not resolvers should begin to go in a util folder